Library/Formula/moe.rb

```diff
@@ -0,0 +1,14 @@
+class Moe < Formula
+  homepage "https://www.gnu.org/software/moe/moe.html"
+  url "http://ftpmirror.gnu.org/moe/moe-1.7.tar.lz"
```

Comment:
Please add `mirror "https://ftp.gnu.org/pub/gnu/moe/moe-1.7.tar.lz"` below this line